frontend/src/components/utils/Search.tsx

```diff
@@ -259,6 +275,7 @@ const Search: React.FC<Props> = React.memo(({ dashboard }) => {
             autoCapitalize="none"
             onChange={onChange}
             compact={compact}
+            defaultValue={value}
```

Comment:
Shouldn't it be `query` instead of `value`?

```suggestion
            defaultValue={query}
```